common/changes/office-ui-fabric-react/person-edge-polish_2017-11-06-16-27.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Persona: Flex sizing for presence with custom coinSize, DirectionalHint for Tooltip location, polished example spacing and documentation on PersonaPage.",
+      "type": "patch"
```

Comment:
Considering this is adding a new capability to the Persona component, I would consinder this a minor version, not a patch

Reply:
With moving 'Flex sizing for presence with custom coinSize' to a separate PR, I think the rest of the changes would fall under the category of 'visual bug fixes', thus not requiring a minor version bump. What do you think?